lib/reek/cli/input.rb

```diff
@@ -1,7 +1,7 @@
 require 'reek/source'
 
 module Reek
-  module CLI
+  module Cli
```

Comment:
Alright, then let's leave it like it is and fix the spelling in a separate PR some time later.